lib/yt/collections/reports.rb

```diff
@@ -188,7 +189,7 @@ def reports_params
           params['end-date'] = @days_range.end
           params['metrics'] = @metrics.keys.join(',').to_s.camelize(:lower)
           params['dimensions'] = DIMENSIONS[@dimension][:name] unless @dimension == :range
-          params['include-historical-channel-data'] = 'true'
+          params.merge!({'include-historical-channel-data' => @historical}) if @historical
```

Comment:
@kangkyu Follow the other lines and avoid `merge!` since this might have side-effects if other methods call this code again:

```ruby
params['include-historical-channel-data'] = @historical if @historical
```